ui/pages/swaps/swaps.util.js

```diff
@@ -101,12 +101,12 @@ const QUOTE_VALIDATORS = [
   {
     property: 'sourceToken',
     type: 'string',
-    validator: isValidAddress,
+    validator: isValidHexAddress,
```

Comment:
This should be following the A strategy, but I guess that would require creating a local helper function to set the option, since it's passed in as a value :thinking: 

Seems fine to leave it as-is if we're confident the data here is always prefixed though, since its our API.

Reply:
just to be super safe, I could do `validator: (input) => isValidHexAddress(input, { allowNonPrefixed: false })` ?

Reply:
That works!

Reply:
Done in latest